test/ansible/molecule/default/kustomize.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Build kustomize testing overlay
+  # load_restrictor must be set to none so we can load patch files from the default overlay
+  command: '{{ kustomize }} build  --load_restrictor none .'
```

Comment:
Thee CI is failing here. If worked locally and not in the CI is because you have the  `kustomize` locally installed and can perform it directly. But in the CI it will have not.